hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -873,6 +873,22 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.datanode.reconcile.blocks.batch.size</name>
+  <value>1000</value>
+  <description>Run reconcile to checkAndUpdate with batch,
```

Comment:
can you add some recheck the descriptions for both the configs, the first line in. both is same, and doesn't make sense to me, recheck once.

Reply:
Changed in latest PR.